docs/eigenlayer/restaking-guides/0-restaking-user-guide/testnet/rewards-claiming/rewards-claiming-faq.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 2
+title: Rewards Claiming FAQ
+---
+
+
+
+#### When can I claim my rewards?
+
+After a root is posted, rewards are claimable after an activation delay. On mainnet this delay is 1 week and on testnet it is 2 hours.
+
+#### What portion of rewards goes to my operator?
+
+Operators get a fixed 10% portion rewards. 
```

Comment:
Noted in the slack thread, would be good to note this will be changed in near future to be variable